addons/website_forum/views/website_forum.xml

```diff
@@ -444,27 +460,59 @@
     <!-- First, list all forums (those with descriptions first) -->
     <t t-foreach="sorted_forums" t-as="forum">
         <t t-set="has_desc" t-value="forum.description"/>
-        <div t-attf-class="mb-3 col-sm-6 #{not has_desc and 'col-md-3'}">
-            <div t-attf-class="#{has_desc and 'row' or 'd-flex flex-column h-100'} #{no_description_exist and has_desc and 'py-4 bg-200'}">
-                <div t-attf-class="#{has_desc and 'col-md-4 pr-md-0' or 'flex-lg-grow-1'}">
+        <div t-attf-class="#{col_class}">
+            <div class="row py-4 bg-200 mx-1 h-100 o_forum_row">
+                <div t-attf-class="#{img_class}">
                     <a t-attf-href="/forum/#{slug(forum)}">
-                        <div t-attf-class="oe_img_bg rounded shadow-sm py-4 px-5 o_wforum_forum_card_bg flex-shrink-0 h-100 o_not_editable {not has_desc and 'h-100'}"
-                            t-attf-style="#{forum.image_256 and ('background-image: url(%s);' % website.image_url(forum, 'image_256'))} background-position: center;"/>
+                        <div t-attf-class="oe_img_bg rounded shadow-sm o_wforum_forum_card_bg flex-shrink-0 h-100 o_editable #{not forum.image_1920 and not editable and 'opacity-0'}"
+                        t-field="forum.image_1920" t-options="{'widget': 'image', 'class': 'w-100 h-100 o_object_fit_cover'}" />
```

Comment:
already discuss about this 1920 no?

Reply:
I posted a question under your review about this